CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 - Added: Support for `Armenian Language` [[@hovikman][]]
 - Added: `Faker.Vehicle` [[@daytonn][]]
 - Added: Elixir 1.9 support [[@igas][]]
+* Added: Category `Finance` [[@thomascharbonnel][]]
```

Comment:
Since you're first contributor, you must include your name in the contributors area at the end of this file.

Reply:
@vbrazo Thank you, updated now :)